# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../hostap-utils/install.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN hostap-utils-0.4.7-orig/Makefile hostap-utils-0.4.7/Makefile
--- hostap-utils-0.4.7-orig/Makefile	2005-01-01 07:42:51.000000000 +0100
+++ hostap-utils-0.4.7/Makefile	2010-08-04 18:01:59.824090755 +0200
@@ -1,3 +1,6 @@
+DESTDIR =
+prefix = /usr/local
+
 ifndef CFLAGS
 CFLAGS = -O2 -Wall
 endif
@@ -5,7 +8,9 @@
 # Include directory for CVS version
 CFLAGS += -I../driver/modules
 
-all: prism2_srec hostap_crypt_conf hostap_diag hostap_io_debug hostap_rid
+BIN = prism2_srec hostap_crypt_conf hostap_diag hostap_io_debug hostap_rid
+
+all: $(BIN)
 
 prism2_srec: prism2_srec.o util.o
 
@@ -28,6 +33,9 @@
 
 hostap_rid.o: hostap_rid.c util.h
 
+install: all
+	for x in $(BIN); do install -v $$x $(DESTDIR)$(prefix)/bin/$$x; done
+
 clean:
 	rm -f prism2_srec hostap_crypt_conf hostap_diag hostap_rid \
 		hostap_io_debug *.o
